tests/unit/_drivers/test_nidcpower.py

```diff
@@ -44,7 +44,7 @@ def test___single_session_info___initialize_nidcpower_session___session_created(
     session = create_mock_nidcpower_session()
     session_new.side_effect = [session]
 
-    with reservation.initialize_nidcpower_session() as session_info:
+    with reservation.initialize_nidcpower_session(options={}) as session_info:
```

Comment:
The default values are overridden in `initialize_nidcpower_session` since we are using `.env`, so passing default values explicitly here will ensure `.env` values are not used.